Voice Assistant.js

```diff
@@ -0,0 +1,107 @@
+let personName = " ";
+let toDoList = [];
+
+function getReply(command) {
+  if (command.startsWith("Hello my name is")) {
+    const name = command.split("Hello my name is ")[1].trim();
+    if (personName === name) {
+      return `We have met before, ${name}`;
+    } else {
+      personName = name;
+      return `Nice to meet you ${name}`;
+    }
+  } else if (command === "What is my name?") {
+    if (personName) {
```

Comment:
This test will always be true since you initialized the variable with a character (`personName = " "`).

Reply:
This returned false,

Reply:
If the first command called is `What is my name?`, this what will happen:
<img width="223" alt="image" src="https://github.com/oyelajakenny/HYF-Homework/assets/5467758/ba964061-0d78-45ac-b4ac-388fceb1c712">
 
The expected result would be `No name`, or in your implementation `I don't know your name yet`. This is because you have initialised `personName` with a space character. See my previous comment: https://github.com/oyelajakenny/HYF-Homework/pull/6/files#r1659336829